specifyweb/frontend/js_src/lib/components/FormMeta/index.tsx

```diff
@@ -161,7 +161,7 @@ function MetaDialog({
                 <EditHistory resource={resource} />
               </ProtectedAction>
             </ProtectedTool>
-            {isTreeResource(resource) && (
+            {isTreeResource(resource) && !resource.isNew() && (
```

Comment:
Please try to avoid using the `&&` hack when there is more than one condition and prefer explicit ternary in cases like this.
The `&&` hack is fine if there is just one condition, but otherwise it get's a bit too hard to comprehend